src/Integration.UnitTests/Suppression/SonarQubeIssuesProviderTests.cs

```diff
@@ -198,7 +204,7 @@ public void GetIssues_NoIssuesOnServer_ReturnsEmptyList()
             VerifyServiceGetIssues(Times.Exactly(1), "sqkey");
 
             // 2. SonarQube project key doesn't match -> no issues
-            var matches = issuesProvider.GetSuppressedIssues("any project", "any file");
+            var matches = issuesProvider.GetSuppressedIssues("any project", "c:\\anyfile");
```

Comment:
Miscellaneous: changing the setup not to be `null` executed a different code path, which caused an assertion in the product code. Passing an absolute path avoids the assertion.